graphql_compiler/compiler/emit_sql.py

```diff
@@ -205,25 +289,66 @@ def _set_group_by(self, group_by):
         """Set output columns for the fold object."""
         self._group_by = group_by
 
-    def _construct_fold_joins(self, edge, from_alias, to_alias):
+    def _construct_fold_joins(self):
         """Use the edge descriptors to create the join clause between the tables in the fold."""
-        join_clause = sqlalchemy.join(
-            from_alias,
-            to_alias,
-            onclause=(from_alias.c[edge.from_column] == to_alias.c[edge.to_column])
-        )
+        # Start the join clause with the from_table of the first join descriptor
+        _, join_clause, _ = self.join_info[0]
```

Comment:
How do we know there is at least one thing in `join_info`? What happens if there is more than one?